COLLABORATOR_GUIDE.md

```diff
@@ -416,14 +416,15 @@ longer be used.
 
 Node.js uses three Deprecation levels:
 
-* *Documentation-Only Deprecation* refers to elements of the Public API that are
-  being staged for deprecation in a future Node.js major release. An explicit
-  notice indicating the deprecated status is added to the API documentation
-  but no functional changes are implemented in the code. There will be no
-  runtime deprecation warnings emitted for such deprecations by default.
-  Documentation-only deprecations may trigger a runtime warning when Node.js
-  is started with the [`--pending-deprecation`][] flag or the
-  `NODE_PENDING_DEPRECATION=1` environment variable is set.
+* *Documentation-Only Deprecation* refers to elements of the Public API that
+  should be avoided by developers and that might be staged for deprecation in a
+  future Node.js major release. An explicit notice indicating the deprecation
+  status is added to the API documentation but no functional changes are
+  implemented in the code. There will be no runtime deprecation warnings emitted
+  for such deprecations at runtime by default. Documentation-only deprecations
+  may trigger a runtime warning when Node.js is started with the
+  [`--pending-deprecation`][] flag or the `NODE_PENDING_DEPRECATION=1`
```

Comment:
Nit: maybe it is worth to also linkify `NODE_PENDING_DEPRECATION=1` and add the bottom reference:
```md
[`NODE_PENDING_DEPRECATION=1`]: doc/api/cli.md#node_pending_deprecation1
```